src/blueprint/into_schema.rs

```diff
@@ -70,12 +71,17 @@ fn to_type(def: &Definition) -> dynamic::Type {
 
                                         let const_value =
                                             expr.eval(ctx, &Concurrent::Sequential).await?;
-
                                         let p = match const_value {
-                                            ConstValue::List(a) => FieldValue::list(a),
-                                            a => FieldValue::from(a),
+                                            ConstValue::List(a) => Some(FieldValue::list(a)),
+                                            a => {
+                                                if a == ConstValue::Null && is_list {
+                                                    FieldValue::NONE
+                                                } else {
+                                                    Some(FieldValue::from(a))
```

Comment:
`try_from` here is just a blanket implementation that uses `from` thus it's infallible

Reply:
removed try_from as clippy was complaining